packages/@dcl/inspector/src/components/EntityInspector/MaterialInspector/Texture/Texture.tsx

```diff
@@ -0,0 +1,71 @@
+import { useCallback, useRef } from 'react'
+import { useDrop } from 'react-dnd'
+import cx from 'classnames'
+
+import { Block } from '../../../Block'
+import { SelectField } from '../../SelectField'
+import { TextField } from '../../TextField'
+import { Container } from '../../../Container'
+import { Props, Texture, TEXTURE_TYPES, WRAP_MODES, FILTER_MODES } from './types'
+import { ProjectAssetDrop, getModel } from '../../../../lib/sdk/drag-drop'
+import { AssetNodeItem } from '../../../ProjectAssetExplorer/types'
+import { isModel, isValidTexture } from './utils'
+
+const DROP_TYPES = ['project-asset-gltf']
+
+function TextureInspector({ label, texture, files, getInputProps }: Props) {
+  const inputRef = useRef<HTMLInputElement>(null)
+
+  const changeValue = useCallback((value: AssetNodeItem) => {
+    const set = Object.getOwnPropertyDescriptor(window.HTMLInputElement.prototype, 'value')?.set
+    const inputEvent = new Event('input', { bubbles: true })
+
+    set?.call(inputRef.current, value.asset.src)
+    inputRef.current?.dispatchEvent(inputEvent)
+  }, [])
+
+  const [{ isHover }, drop] = useDrop(
+    () => ({
+      accept: DROP_TYPES,
+      drop: ({ value, context }: ProjectAssetDrop, monitor) => {
+        if (monitor.didDrop()) return
+        const node = context.tree.get(value)!
+        const model = getModel(node, context.tree, isModel)
+        if (model) changeValue(model)
+      },
+      canDrop: ({ value, context }: ProjectAssetDrop) => {
+        const node = context.tree.get(value)!
+        return !!getModel(node, context.tree, isModel)
+      },
+      collect: (monitor) => ({
+        isHover: monitor.canDrop() && monitor.isOver()
+      })
+    }),
+    [files]
+  )
+
+  const type = getInputProps(`${texture}.type`)
+  const src = getInputProps(`${texture}.src`)
+  const isValid = isValidTexture(src.value, files)
+
+  return (
+    <Container label={label} className={cx(label, { hover: isHover })} initialOpen={false}>
+      <Block>
+        <SelectField label="Type" options={TEXTURE_TYPES} {...type} />
+      </Block>
+      <Block ref={drop} error={!isValid}>
+        {type.value === Texture.TT_TEXTURE && <TextField ref={inputRef} label="Path" type="text" {...src} />}
+        {/* {type.value === Texture.TT_AVATAR_TEXTURE && <TextField label="User ID" {...getInputProps(`${texture}.userId`)} />}
+        {type.value === Texture.TT_VIDEO_TEXTURE && <TextField label="Video player entity" {...getInputProps(`${texture}.videoPlayerEntity`)} />} */}
```

Comment:
Are these intended? Should we delete them?

Reply:
yeah, this are ok, just left them for when we have a spec for them